torch/_meta_registrations.py

```diff
@@ -2805,6 +2805,26 @@ def meta__foreach_addcop__scalar(self, tensor1, tensor2, scalar=1):
     )
 
 
+@register_meta(
+    [
+        aten._foreach_lerp_.Scalar,
+    ]
+)
+def meta__foreach_lerp__scalar(self, tensor1, scalar=1):
+    torch._check(
```

Comment:
This can just reuse existing functionality like my https://github.com/pytorch/pytorch/pull/104877/files#diff-6dffed1ade0ba3e887f9a4eafa3bfcec267ab2365b8adcb91bd391f49b3fd2e3R2789